src/app/api/auth/google/callback/route.ts

```
@@ -54,42 +68,62 @@ export async function GET(req: NextRequest) {
       }
     }
 
+    // Create authenticated session
     const session = await lucia.createSession(userId, {});
     createSessionCookie(session.id);
   } catch (error: any) {
     console.error("api/auth/google/callback: error", error.message);
     return new Response("Internal server error", { status: 500 });
   } finally {
+    // Redirect to default URL regardless of success/failure
     return NextResponse.redirect(new URL(DEFAULT_REDIRECT_URL, req.url));
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)